SnapXEats/SnapXEats/Modules/RestaurantDirections/View/RestaurantDirectionsViewController.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  RestaurantDirectionsViewController.swift
+//  SnapXEats
+//
+//  Created by synerzip on 28/02/18.
+//  Copyright © 2018 SnapXEats. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import MapKit
+
+// To Distingugish between Restaurant marker and Current Location Marker the
+class CurrentLocationAnnotation: MKPointAnnotation {
+}
+
+class RestaurantDirectionsViewController: BaseViewController, StoryboardLoadable {
+
+    private let locationTitleLeftInsetMargin: CGFloat = 15.0
```

Comment:
move this constant in constant file. with some decent names

Reply:
This is Specific to View Controller so kept here. Not needed elsewhere in the app

Reply:
Move  then in enum at top of class and give some better name then